### ISSUE TITLE
Add a link from Drive -> Dashboard

### ISSUE THREAD

**Issue by therealjeffg** [fission-suite/drive#138]
We should figure out the right spot to link from Drive back to Dashboard. My vote would be turning the username displayed at the top left into a link that points to dashboard, or maybe having a 'gear' icon + 'settings' in the hotdog menu at the top right.

**Comment by matheus23** [comment]
I think there was some ideas around this from @bmann, where we could have a common "navigation" component to go from app to app.

I'm not sure the following is exactly what he was thinking of, so I guess here is my brainstorming about that:
1. The navigation component should be configurable for each user, probably in the dashboard.
2. By default, some main fission apps should be linked: Auth lobby, Drive, Dashboard.

And it should probably look the same for every app. Maybe there's a webnative feature for listing the apps configured for the current user, and maybe there's some fission-kit stuff in there for having a consistent navigation component.

Such a thing could subsume this feature, but might be some steps ahead of this and something for later probably. Just throwing it in there for cross-pollination :P

**Comment by bmann** [comment]
I think I was thinking more “network” at the time — all your Fission apps available to you. 

This would be mirrored in a native mobile app. 

But yeah very much a much bigger idea and not even sure it fits the thinking for people. A list of apps in Drive might be the thing to do, but that also is making some choices about Drive as platform vs product. 

Good to capture here at least ;)

**Comment by therealjeffg** [comment]
I don't think we need to overthink this or go for a general solution - instead I think it's maybe a best practice for fission-based apps to link back to our dashboard, and we should reflect this best practice in Drive.

**Comment by bmann** [comment]
Yep, agreed @therealjeffg. I think that link back to the Dashboard should be to ... "do some thing". Most of this is around security / account management, like managing permissions, setting up recovery, etc.
